qiita_db/meta_util.py

```diff
@@ -118,6 +116,40 @@ def get_accessible_filepath_ids(user):
                              in SampleTemplate(study_id).get_filepaths()]
             filepath_ids.update(sample_fp_ids)
 
+    # Next, the public processed data
+    processed_data_ids = ProcessedData.get_by_status('public')
+    for pd_id in processed_data_ids:
```

Comment:
I'm just following the pattern on this function. I already noticed that this function is quite bad and needs some refactoring, but I think it needs to be done outside of this PR. See https://github.com/biocore/qiita/issues/1010